lib/vsc/install/shared_setup.py

```diff
@@ -121,6 +122,7 @@ def _log(self, level, msg, args):
 
 # available authors
 ag = ('Andy Georges', 'andy.georges@ugent.be')
+asg = ('Álvaro Simón García', 'alvaro.simongarcia@UGent.be')
```

Comment:
I'm not sure if this will work with latin-1... :smile: maybe we should remove the accents